app/javascript/src/Stats/inlinechart/InlineChart.scss

```diff
@@ -0,0 +1,67 @@
+#mini-charts {
```

Comment:
These styles the ones from removed file `app/assets/stylesheets/provider/_charts.scss`.